gammapy/data/tests/test_data_store.py

```diff
@@ -58,6 +58,16 @@ def test_DataStore_load(data_manager):
     events = data_store.load(obs_id=23037, filetype='events')
     assert isinstance(events, EventList)
 
+@pytest.mark.xfail
+@requires_data('hess')
+def test_DataStore_load_all(data_manager):
+    """Test loading data and IRF files via the DataStore"""
+    data_store = data_manager['hess-hap-hd-prod01-std_zeta_fullEnclosure']
+
+    events_list = data_store.load_all(filetype='events')
```

Comment:
Rename `events_list` -> `event_lists`.

For assert, how about accessing a header keyword and or the first event energy or really any property from the first and last event list? This would make sure that the correct data is loaded. Something like:
```
event_lists[0]['ENERGY'][0] == ...
event_lists[-1]['ENERGY'][0] == ...
```